spec/rototiller/utilities/env_var_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+require 'pry'
+
+describe EnvVar do
+
+  @count = 0
+  ['with_default', 'without_default'].each do |method_signature|
+
+    context method_signature do
+
+      ['ENV set', 'ENV not set'].each do |env_set|
+
+        context env_set do
+
+          let (:var_name)       { "VARNAME_#{(0...8).map { (65 + rand(26)).chr }.join}" }
+          let (:var_message)    { "This is how you use #{var_name}" }
+          let (:var_default)    { "VARDEFAULT_#{(0...8).map { (65 + rand(26)).chr }.join}" }
+          let (:var_env_value)  { "VARENVVALUE_#{(0...8).map { (65 + rand(26)).chr }.join}" }
```

Comment:
this is cool.  i should use this randomization for task names in RototillerTask tests